doctr/models/detection/zoo.py

```diff
@@ -10,25 +10,22 @@
 
 __all__ = ["detection_predictor"]
 
-default_cfgs: Dict[str, Dict[str, Any]] = {
-    'db_resnet50': {'model': 'db_resnet50', 'post_processor': 'DBPostProcessor'},
-    'linknet': {'model': 'linknet', 'post_processor': 'LinkNetPostProcessor'},
-}
+archs = ['db_resnet50', 'linknet']
 
 
 def _predictor(arch: str, pretrained: bool, **kwargs: Any) -> DetectionPredictor:
 
-    if default_cfgs.get(arch) is None:
+    if arch not in archs:
         raise ValueError(f"unknown architecture '{arch}'")
 
     # Detection
-    _model = detection.__dict__[default_cfgs[arch]['model']](pretrained=pretrained)
+    _model = detection.__dict__[arch](pretrained=pretrained)
     kwargs['mean'] = kwargs.get('mean', _model.cfg['mean'])
     kwargs['std'] = kwargs.get('std', _model.cfg['std'])
     predictor = DetectionPredictor(
         DetectionPreProcessor(output_size=_model.cfg['input_shape'][:2], **kwargs),
         _model,
-        detection.__dict__[default_cfgs[arch]['post_processor']]()
+        _model.postprocessor
```

Comment:
even though, it won't be a double instantiation because we pass it by reference here, if you print the predictor, you will see 2 postprocessors. I think we have 2 options here:
1. not passing the postprocessor to the predictor and in the predictor's call, it will use `self.model.postprocessor`
2. in model, only the class (not an instance) is set to `self.postprocessor` so we pass it in the predictor but using the preprocessor's constructor

The easiest would probably be the first option

Reply:
yes